txgh/lib/txgh/github_api.rb

```diff
@@ -34,17 +34,21 @@ def create_ref(branch, sha)
       client.create_ref(repo_name, branch, sha) rescue false
     end
 
-    def update_contents(branch, content_map, message)
-      content_map.each do |path, new_contents|
+    def update_contents(branch, content_list, message)
+      content_list.each do |file_params|
+        path = file_params.fetch(:path)
+        new_contents = file_params.fetch(:contents)
```

Comment:
Yeah, it's used by the [`ResourceCommitter`](https://github.com/lumoslabs/txgh/blob/055eccdacc3c335e4a53086ed68ecd2f771c23c7/txgh/lib/txgh/resource_committer.rb#L21).